person.rb

```diff
@@ -18,4 +23,17 @@ def can_use_services?
   def of_age?
     age >= 18
   end
+
+  public
+
+  def correct_name
+    @name
+  end
 end
+
+person = Person.new(22, 'maximilianus')
+p person.correct_name
+capitalized_person = capitalize_decorator.new(person)
+p capitalized_person.correct_name
+capitalized_trimmed_person = trimmer_decorator.new(capitalized_person)
```

Comment:
OPTIONAL
- [ ] Similarly, change `trimmer_decorator` to `TrimmerDecorator`

Reply:
@rloterh It gave errors that's why I opted to change 

![Capture](https://user-images.githubusercontent.com/61447440/180058706-ddca7bc2-6ef4-49cd-a35f-5d04c138ebbd.PNG)

Reply:
It's okay if it's working other way round for you! 😄